applications/plugins/SofaCarving/examples/SimpleCarving.py

```diff
@@ -0,0 +1,100 @@
+import Sofa
+
+
+#Choose in your script to activate or not the GUI
+USE_GUI = True
+
+
+def main():
+    import SofaRuntime
+    import Sofa.Gui
+
+    root = Sofa.Core.Node("root")
+    createScene(root)
+    Sofa.Simulation.init(root)
+
+    if not USE_GUI:
+        for iteration in range(10):
+            Sofa.Simulation.animate(root, root.dt.value)
+    else:
+        Sofa.Gui.GUIManager.Init("myscene", "qglviewer")
+        Sofa.Gui.GUIManager.createGUI(root, __file__)
+        Sofa.Gui.GUIManager.SetDimension(1080, 1080)
+        Sofa.Gui.GUIManager.MainLoop(root)
+        Sofa.Gui.GUIManager.closeGUI()
+
+def createScene(root):
+    
+    root.gravity=[0, 0, 0]
+    root.dt=0.05
+    root.showBoundingTree = 0
+    root.addObject('RequiredPlugin', name="plug1", pluginName="Sofa.Component.Collision.Detection.Algorithm Sofa.Component.Collision.Detection.Intersection Sofa.Component.Collision.Geometry Sofa.Component.Collision.Response.Contact")
+    root.addObject('RequiredPlugin', name="plug2", pluginName="Sofa.Component.Constraint.Projective Sofa.Component.LinearSolver.Iterative Sofa.Component.ODESolver.Backward")
+    root.addObject('RequiredPlugin', name="plug3", pluginName="Sofa.Component.Engine.Select Sofa.Component.IO.Mesh Sofa.Component.Mass ")
+    root.addObject('RequiredPlugin', name="plug4", pluginName="Sofa.Component.Mapping.Linear Sofa.Component.Mapping.NonLinear Sofa.Component.SolidMechanics.FEM.Elastic Sofa.Component.StateContainer")
+    root.addObject('RequiredPlugin', name="plug5", pluginName="Sofa.Component.Topology.Container.Dynamic Sofa.Component.Topology.Mapping")
+    root.addObject('RequiredPlugin', name="plug6", pluginName="Sofa.Component.Visual Sofa.GL.Component.Rendering3D")
+    root.addObject('RequiredPlugin', name="plug7", pluginName="Sofa.Component.Collision.Detection.Algorithm SofaCarving")
```

Comment:
We should really decide a rule for that because all the scene in examples have been updated without adding them in a specific Node.
Each time adding them in or outside a node add some changes in the history...

Reply:
In stlib we are doing:
```python
from stlib.scene import Scene
def createScene(root): 
       scene = Scene(root, plugins=["plugin1", "plugin2] 
```

But if you don't want dependency ot STLIB... make your own 
```python
def Scene(root, plugins) 
    plugins = root.addChild("Plugins")
    for plugin_name in plugins:
         plugins.addObject("RequiredPlugin", name=plugin_name)
    return root     
```